backend/editor/entries.py

```diff
@@ -614,9 +614,13 @@ async def delete_taxonomy_project(self, branch, taxonomy_name):
         Delete taxonomy projects
         """
 
-        query = f"""
-            MATCH (n:{self.project_name}) DETACH DELETE n RETURN count(n)
+        delete_query = """
+            MATCH (n:PROJECT {taxonomy_name: $taxonomy_name, branch: $branch})
+            DELETE n
         """
-        result = await get_current_transaction().run(query)
-        count = await result.single()
-        return count[0]
```

Comment:
Hey @alexgarel 

I figured out what the issue was. It was in query and also in creating dummy data.

I have resolved it.

Reply:
kudos